static/panes/editor.ts

```diff
@@ -595,7 +595,8 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
         };
 
         const getEditorConfig = () => {
-            return Components.getEditor();
+            // TODO(jeremy-rifkin): Can this.settings.defaultLanguage really be undefined?
```

Comment:
probably depends on the order of initialization, so in theory yes, but then something with the order of things should probably be fixed